app/views/families/index.html.erb

```diff
@@ -29,15 +29,15 @@
   </div>
 
   <div class="tab inactive-families">
-    <p class="title"><%= t(".not_active_families") %></p>
+    <p class="title"><%= t("families.index.inactive_families") %></p>
```

Comment:
I believe you can use relative loading of a translation here: i.e. `t(".inactive_families")`